runelite-client/src/main/java/net/runelite/client/plugins/inventorygrid/InventoryGridOverlay.java

```diff
@@ -74,6 +72,22 @@ public Dimension render(Graphics2D graphics)
 		final Widget if1DraggingWidget = client.getIf1DraggedWidget();
 		final Widget inventoryWidget = client.getWidget(WidgetInfo.INVENTORY);
 
+
+		for (int i = 0; i < INVENTORY_SIZE; ++i)
+		{
+
+			WidgetItem targetWidgetItem = inventoryWidget.getWidgetItem(i);
+
+			final Rectangle bounds = targetWidgetItem.getCanvasBounds();
+
+			if (config.showIndividualGrid())
+			{
+				graphics.setColor(config.colorGrid());
+				graphics.fill(bounds);
```

Comment:
This is incorrect because if both showIndividualGrid (which maybe should be renamed to something with a better name?) and showGrid is on, it will double-fill the slot with the configured color, which if using alpha will make the end color incorrect.

Reply:
'it will double-fill the slot with the configured color, which if using alpha will make the end color incorrect.' ye I kept that as I thought it looked nice